Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+PRINT 'Begin Sql scripts execution'
+SET XACT_ABORT ON
+SET NOCOUNT ON
+
+DECLARE @_acquireLock VARCHAR(64)
+DECLARE @_acquireLockMilliseconds INT
+DECLARE @_acquireLockMaxAttempts INT
+DECLARE @_longRunningThresholdMilliSeconds INT
+DECLARE @_lastAttemptsToKillSessions INT
+DECLARE @_isHosted INT
+
+PRINT 'Adding SQL files into temporary table _tmpSqlFilesTable'
+
+IF OBJECT_ID (N'_tmpSqlFilesTable', N'U') IS NOT NULL 
+BEGIN
+   PRINT 'Temporary table exists already. Dropping table _tmpSqlFilesTable'
+   DROP TABLE _tmpSqlFilesTable
```

Comment:
>DROP TABLE _tmpSqlFilesTable [](start = 3, length = 28)

1. Wouldn't it create a problem with parallel executions? one execution deletes the table created by another execution, if not done in transaction.
2. Do it in a TRY CATCH and provide proper error to end user in case this DROP statement fails